infra/core/storage/storage-account.bicep

```diff
@@ -38,6 +39,7 @@ resource storage 'Microsoft.Storage/storageAccounts@2022-05-01' = {
       defaultAction: 'Allow'
     }
     publicNetworkAccess: publicNetworkAccess
+    supportsHttpsTrafficOnly: requireHttpsConnection
```

Comment:
You could just set it to true, but I have a hard time deciding what to parameterize.